charts/kubewarden-controller/values.yaml

```diff
@@ -21,6 +21,13 @@ fullnameOverride: ""
 # Secrets to pull container images from private registries
 imagePullSecrets: []
 
+# -- Additional labels to add to all resources
+additionalLabels: {}
+  # app: kubewarden-controller
+
+# -- Additional annotations to add to all resources
+additionalAnnotations: {}
```

Comment:
```suggestion
```
I would prefer to drop this from here, as we aren't templating for it in this PR, and people may think that it is honored when it isn't. I happily welcome the templating for it in this PR or a follow-up though.